Rebrand concept docs and guides

marklewin opened the pull request:
## Description

Change Nexmo -> Vonage in the guides and concept docs.

Took care to leave URLs, Nexmo Call Control Objects and Nexmo CLI unchanged.

[DEVX-4717](https://nexmoinc.atlassian.net/browse/DEVX-4717)

tbedford commented on _documentation/en/client-sdk/setup/create-your-application.md:
```diff
@@ -8,20 +8,20 @@ navigation_weight: 1
 
 All of these may be [created by your backend](/conversation/overview). 
-If you wish to get started and experience using the SDK without any implementation of your backend, this tutorial will show you how to do so, using the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli).
+If you wish to get started and experience using the SDK without any implementation of your backend, this tutorial will show you how to do so, using the [Nexmo CLI](https://github.com/Vonage/nexmo-cli).
```
I get a 404 for https://github.com/Vonage/nexmo-cli

tbedford commented on _documentation/en/concepts/guides/glossary.md:
```diff
@@ -44,7 +44,7 @@ For more information on managing CDRs see the [Redact API documentation](/redact
 Nexmo Command Line Interface (CLI) tool. 
 
-Further information can be found in the [CLI repository](https://github.com/Nexmo/nexmo-cli).
+Further information can be found in the [CLI repository](https://github.com/Vonage/nexmo-cli).
```
I get a 4040 for https://github.com/Vonage/nexmo-cli

tbedford commented on _documentation/en/conversation/guides/jwt-acl.md:
```diff
@@ -76,11 +76,11 @@ In the previous section, you can see that the `acl` claim has `paths` object con
 ### CLI
 
-Currently, you can use the beta version of the [Nexmo CLI](https://github.com/Nexmo/nexmo-cli/tree/beta) to create a JWT [including the appropriate claims](https://github.com/Nexmo/nexmo-cli/tree/beta#jwt)
+Currently, you can use the beta version of the [Nexmo CLI](https://github.com/Vonage/nexmo-cli/tree/beta) to create a JWT [including the appropriate claims](https://github.com/Vonage/nexmo-cli/tree/beta#jwt)
```
404 for https://github.com/Vonage/nexmo-cli/tree/beta

tbedford commented on _documentation/en/conversation/guides/jwt-acl.md:
```diff
@@ -89,7 +89,7 @@ nexmo jwt:generate ./private.key sub=jamie exp=$(($(date +%s)+86400)) acl='{"pat
 ### Node
 
-The beta version of the [Nexmo Node](https://github.com/Nexmo/nexmo-node/tree/beta#jwt) library can also create a JWT [including the appropriate claims](https://github.com/Nexmo/nexmo-node/tree/beta#jwt)
+The beta version of the [Vonage Node](https://github.com/Vonage/nexmo-node/tree/beta#jwt) library can also create a JWT [including the appropriate claims](https://github.com/Vonage/nexmo-node/tree/beta#jwt)
```
404 for https://github.com/Vonage/nexmo-cli/tree/beta

tbedford replied:
Library -> Server SDK

tbedford commented on _documentation/en/conversation/guides/jwt-acl.md:
```diff
@@ -76,11 +76,11 @@ In the previous section, you can see that the `acl` claim has `paths` object con
 It is not necessary to provide users with the permissions to access all of these paths. For instance, if a user was not going to upload or receive push notifications, you could create a JWT without including the `/*/applications/**`or `/*/push/**` paths.
 
-## Nexmo Client Libraries
+## Vonage Client Libraries
```
Vonage Server SDKs

tbedford commented on _documentation/en/conversation/guides/jwt-acl.md:
```diff
@@ -106,7 +106,7 @@ const aclPaths = {
 }
 
-Nexmo.generateJwt(PRIVATE_KEY, {
+Vonage.generateJwt(PRIVATE_KEY, {
```
Has the class name definitely changed?

marklewin replied:
No ... good spot

tbedford commented on _documentation/en/conversation/guides/jwt-acl.md:
```diff
@@ -117,14 +117,14 @@ Nexmo.generateJwt(PRIVATE_KEY, {
 ### PHP
 
-The current version of the [Nexmo PHP](https://github.com/Nexmo/nexmo-php) library can also create a JWT including the appropriate claims when using the Keypair authentication.
+The current version of the [Vonage PHP](https://github.com/Vonage/nexmo-php) library can also create a JWT including the appropriate claims when using the Keypair authentication.
```
Library -> Server SDK

tbedford replied:
https://github.com/Vonage/nexmo-php redirects to https://github.com/Vonage/vonage-php-sdk-core

marklewin commented:
Adding DO NOT MERGE because I want to check that this PR also addresses the issues raised in [DEVX-3912](https://nexmoinc.atlassian.net/browse/DEVX-3912)

marklewin commented:
Will deal with DEVX-3912 changes in a new branch. Marking as ready for review/merge.

tbedford commented on _documentation/en/concepts/guides/authentication.md:
```diff
@@ -142,9 +142,9 @@ An example of generating a JWT for a Client SDK application is as follows:
+More examples of generating JWTs with ACLs can be found in the [Vonage documentation](/conversation/guides/jwt-acl).
 
-Further information on the Nexmo CLI can be found in its [repository on GitHub](https://github.com/Nexmo/nexmo-cli).
+Further information on the Nexmo CLI can be found in its [repository on GitHub](https://github.com/Vonage/nexmo-cli).
```
404 for https://github.com/Vonage/nexmo-cli
